### PR TITLE
enable remote image tree

### PR DESCRIPTION
**Pull request by baude**
Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

File: pkg/api/handlers/libpod/images.go [hunk 1]
```diff
@@ -74,14 +79,14 @@ func ImageTree(w http.ResponseWriter, r *http.Request) {
 			errors.Wrapf(err, "failed to parse parameters for %s", r.URL.String()))
 		return
 	}
-
-	tree, err := img.GenerateTree(query.WhatRequires)
+	ir := abi.ImageEngine{Libpod: runtime}
+	options := entities.ImageTreeOptions{WhatRequires: query.WhatRequires}
+	report, err := ir.Tree(r.Context(), name, options)
```
Comment (vrothberg) [inline pkg/api/handlers/libpod/images.go:84]: Can we move the 404 check here?

Comment (baude) [reply pkg/api/handlers/libpod/images.go:84]: sure

**Comment by baude** [comment]
fixed

**Pull request merged**